spec/lib/portus/ldap_spec.rb

```diff
@@ -332,6 +332,15 @@ def load_configuration_test
       ]
     end
 
+    let(:multiple_emails) do
+      [
+        {
+          "dn"     => ["ou=users,dc=example,dc=com"],
+          "email"  => ["user1@example.com","user2@example.com"]
```

Comment:
rubocop is also complaining that you're not leaving the proper alignment for each element. This means that you should leave only one space between the "email" key and the arrow. Then the line above should be aligned to the same level.